comms/natsd/nats_main.go

```diff
@@ -99,5 +99,5 @@ func startServer() {
 		})
 	})
 
-	e.Logger.Fatal(e.Start(":8924"))
+	e.Logger.Fatal(e.Start(":8925"))
```

Comment:
8925 is used by the "discovery / comms" server... NATS / exchange should stay on 8924